shell/bashrc

```diff
@@ -319,7 +324,6 @@ function phpbrew ()
             unset PHPBREW_PATH
             eval `$BIN env`
             __phpbrew_set_path
-            echo "phpbrew is turned off."
```

Comment:
Agreed, but IMHO it's a potential BC break. I suggest to change the target branch to develop, which will be merged and released in next major version (1.23 probably). What do you think?